tutorials/Python/setup-conda-linux.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
This works on `macOS` as well. I suggest to change the name to `setup-conda.sh` (and remove `WTcourse` name from the comment).

Reply:
Cool thanks, I had forgotten I wanted to ask you if we could switch from `Linux` to `Linux and MacOS` :)

Do you think it's worth changing the environment name too? We could name it something like "OxPython"?

Reply:
`OxPython` sounds good! Or `oxpy` for the lazy.

Reply:
Cool thanks I'll update and merge